images/base/Dockerfile

```diff
@@ -23,8 +23,15 @@ ENV LC_ALL=en_US.UTF-8
 # Download in code-server into path. sail will typically override the binary
 # anyways, but it's nice to have this during the build pipepline so we can
 # install extensions.
-RUN wget -O /usr/bin/code-server https://codesrv-ci.cdr.sh/latest-linux && \
-    chmod +x /usr/bin/code-server
 
+# Download and Install code-server: https://github.com/cdr/code-server/releases/tag/3.0.0
+
+RUN wget -O code-server.tgz "https://codesrv-ci.cdr.sh/releases/3.0.1/linux-x86_64.tar.gz"; \
+	tar -C /usr/lib -xzf code-server.tgz; \
+	rm code-server.tgz;
+
+RUN ln -s /usr/lib/code-server-3.0.1-linux-x86_64/code-server /usr/bin/code-server; \
+    chmod +x /usr/lib/code-server-3.0.1-linux-x86_64/code-server; \
+    chmod +x /usr/bin/code-server;
```

Comment:
This should be a single RUN. Also, instead of using `; \`, use `&& \` (because otherwise if any of these fail the build will still succeed)